docs/CONFIG-PROPERTIES.md

```diff
@@ -31,7 +31,9 @@
 | debug.enable.vga | boolean | false | allow VGA console on device |
 | debug.enable.ssh | authorized ssh key | empty string(ssh disabled) | allow ssh to EVE |
 | debug.enable.console | boolean | false | allow console access to EVE (reboot required to disable) |
-| debug.default.loglevel | string | info | min level saved in files on device |
+| debug.default.loglevel | string | info | min level saved in files on device. Used logrus log levels as described here ["https://pkg.go.dev/github.com/sirupsen/logrus"]. |
+| debug.syslog.loglevel | string | info | min level of the syslog messages saved in files on device. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]. Compatibility with logrus log levels ("debug.default.loglevel" field) is also supported, but is not recommended for usage. |
```

Comment:
What does the compatibility sentence mean? That I can use a logrus level string with this knob? Thus "error" instead of "err"?

Reply:
Correct. Logrus levels (names) are different. This is made for the sake of unification, so that user can provide logrus levels for new settings as well. Not sure this is a good idea, but it seems from usability point of view having different loglevel names is even worse.